examples/wgpu-example-compute-runner/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "wgpu-example-compute-runner"
+version = "0.1.0"
+authors = ["charles-r-earp <charles.r.earp@gmail.com>"]
```

Comment:
this feels weird to ask (@repi maybe could provide feedback here?), but should this be embark's email that's in every other crate, or yours?